### ISSUE TITLE
[rotateFont] Segmentation Fault happens when using rotateFont or mergeFont to rename glyph names.

### ISSUE THREAD

**Issue by jimmymasaru** [adobe-type-tools/afdko#1338]
### Environment
macOS 11.2.3
python 3.9.2 
AFDKO installed by `pip3 install afdko`

### Command
```
tx -t1 -n -decid -usefd 0 SourceHanSans-Regular.otf decidfont.ps
rotateFont -t1 -rtf map.txt decidfont.ps gidfont.ps
```
[SourceHanSans-Regular.otf](https://github.com/adobe-fonts/source-han-sans/blob/release/OTF/Japanese/SourceHanSans-Regular.otf)
The contents of `map.txt`
```
.notdef	.notdef	None	0	0
cid1	uni0020	None	0	0
```

**Comment by punchcutter** [comment]
The segfault was introduced in 3.0.0. 2.9.1 still works.

**Comment by punchcutter** [comment]
Introduced in https://github.com/adobe-type-tools/afdko/commit/4ddd28a388c080a044851ed0794a395d1b89e750#diff-013e8e3aefbe9ab19ebd3a80996edc1d753ee5bb3487fef21dccb710d16ddacaR340

**Comment by punchcutter** [comment]
@jimmymasaru What exactly do you want to do here with `-decid`? The real problem here is that it's not possible to make a name-keyed font with this many glyphs so it never should have worked in the first place. It "worked" in older releases by making a bad file, but that was fixed in 3.0.0. If you want to manipulate a particular glyph or range of glyphs you can use the `-g` flag to get only those glyphs.
